### DISCUSSION TITLE
IMG widget with src from data

### DISCUSSION THREAD

**Discussion by andreyzin** [zepp-health/zeppos-forum#37] [Q&A] [answered]
ZEPP OS, API version 1
Is it possible to display image from data folder, not assets? Maybe there is any opportunity to bypass this?

**Comment by htoooth** [comment, marked as answer]
hi @andreyzin  there,
Currently there is no way to load image from data folder in zeppos 1 and 2. We are developing this feature  in os 3.0.

**Reply by chipset2011** [reply]
Will OS 3.0 be available only in new watches, for example GTR5?

**Reply by htoooth** [reply]
hi @chipset2011 :
It depends on company policy.  I don't know as much as you.
